starboard/build/config/BUILDCONFIG.gn

```diff
@@ -358,11 +362,34 @@ template("executable") {
 }
 
 template("shared_library") {
-  target_with_platform_configs(target_name) {
+  original_target_name = target_name
+  target_with_platform_configs(original_target_name) {
```

Comment:
Ideally building the target name would build something that you could run (i.e. in a modular build it would also build the loader), but I'm not sure how to accomplish that with dependencies already in place